app/src/main/java/com/duckduckgo/app/cta/ui/Cta.kt

```diff
@@ -219,11 +182,8 @@ sealed class DaxDialogCta(
     }
 
     companion object {
```

Comment:
These values here seem related to the ones moved to the helper class, but I wonder which is the connection between them and which is the correct place for both. Moving some of them feels to me like removing somehow that connection. Sorry, I'm missing some context here to come up with a suggestion.

Reply:
This has all changed now